lib/oli_web/router.ex

```diff
@@ -513,6 +513,7 @@ defmodule OliWeb.Router do
     delete("/:project/lock/:resource", Api.LockController, :release)
 
     get("/:project/alternatives", Api.ResourceController, :alternatives)
+    get("/:project/is_experiment_enabled", Api.ResourceController, :is_experiment_enabled)
```

Comment:
We do not need this, given that `has_experiments` lives on the project, and can be passed down via context to the page editor

Reply:
Thanks for pointing the field `has_experiments` on the `project` model. I didn't know how to persist the state so I created a table for that. I'll restructure that part to use it then.